includes/Stories_Renderer/Renderer.php

```diff
@@ -507,6 +502,9 @@ public function render_single_story_content() {
 		$single_story_classes = $this->get_single_story_classes();
 		$lightbox_state       = 'lightbox' . $this->current()->get_id();
 
+		// Web Stories Styles for AMP and non-AMP pages.
+		wp_enqueue_style( self::STYLE_HANDLE );
```

Comment:
```suggestion
		$this->enqueue_style( self::STYLE_HANDLE );
```

Reply:
why do we need to use `$this->enqueue_style()`, if we already have the style registered? 

`$this->enqueue_style()` registers the given style first with the dependencies and then enqueues it, but here we have already registered the same script using `$this->register_script()`. Also I believe this adheres to the Embed Block renderer. I took reference from it while fixing the e2e test and updated this.

Reply:
Right, can keep this as `wp_enqueue_style()`

Reply:
Thanks for the detail. Agreed to as is.